Character.cpp

```diff
@@ -86,32 +88,45 @@ int Character::takeDamage(int damage)
     return hitPoints;
 }
 
-
-#include <assert>
 void Character::attackInternal(Character& other)
 {
-    if( other.hitPoints <= 0 )
+    if(other.hitPoints <= 0)
     {
         /*
         When you defeat another Character: 
             a) your stats are restored to their initial value
```

Comment:
This instruction has been revised to:
```
a) your stats are restored to their initial value if they are lower than it.
```
- [ ] revise code according to new instruction